create_tables.sql

```diff
@@ -98,3 +100,19 @@ create table arrange_mode
     armd_is_expire bit(1) default 0,
     armd_is_delete bit(1) default 0
 );
+
+
+DROP TABLE IF EXISTS user_prefer;
+
+create table user_prefer
+(
+        pref_id varchar(14) unique key,
+        user_id int default 0,
```

Comment:
we need to review all table 
I think this problem should be deal next time but not this merge